mgoutil/mgoutil.go

```diff
@@ -0,0 +1,39 @@
+// Package mgoutil provides various methods for working with the mgo.v2 package,
+// implementing various oft-used behaviors
+package mgoutil
+
+import (
+	"github.com/levenlabs/go-llog"
+	"gopkg.in/mgo.v2"
+)
+
+// Index is used in MustEnsureIndexes to convey all the information needed to
+// ensure a background index
+type Index struct {
+	Name string // human readable name for the index
```

Comment:
What's the purpose of the "name"? It seems confusing that its not sent to mongo and that its just a "comment" essentially? Can't we just actually use a comment above where the index is created to name it?